components/script/dom/globalscope.rs

```diff
@@ -2227,6 +2231,66 @@ impl GlobalScope {
         }))
     }
 
+    pub fn create_image_bitmap(
+        &self,
+        image: ImageBitmapSource,
+        options: &ImageBitmapOptions,
+    ) -> Rc<Promise> {
+        let global = self.global();
+        let in_realm_proof = AlreadyInRealm::assert(&global);
+        let p = Promise::new_in_current_realm(&global, InRealm::Already(&in_realm_proof));
+        if options.resizeWidth.unwrap() == 0 {
```

Comment:
The spec says "is present and is 0", which means we should use `options.resizeWidth.map_or(false, |w| w == 0)`, rather than panicking if the resizeWidth value is not defined in JS. See the docs for [map_or](https://doc.rust-lang.org/stable/std/option/enum.Option.html#method.map_or). This should be used for the resizeHeight check too.

Reply:
This is a better way to do it. Thanks. The panicking does not take place anymore.